src/components/ArticleIndividual/ArticleIndividual.jsx

```diff
@@ -0,0 +1,35 @@
+function ArticleIndividual({ loading, currentArticleIndividual }) {
+  function helperFunction() {
+    console.log(currentArticleIndividual, "individual article page");
+  }
+  helperFunction();
+
+  // could refactor to add loading states
```

Comment:
You want an api call here to /api/articles/:article_id - to get the individual article. You can use useParams to grab the article_id from the URL.